### ISSUE TITLE
How to ignore Pod/ Folder from SwiftLint ?

### ISSUE THREAD

**Issue by duongsavvycom** [realm/SwiftLint#1423]
How to ignore Pod/ Folder from SwiftLint ?

I could not find this  question anywhere, so  please help ! Thanks

**Comment by marcelofabri** [comment]
Take a look at the [configuration section on README](https://github.com/realm/SwiftLint#configuration).

```yml
excluded: # paths to ignore during linting. Takes precedence over `included`.
  - Carthage
  - Pods
```

**Comment by alokc83** [comment]
Excluding is not working I have specific set for Pods dir. How should I provide the path

**Comment by guangLess** [comment]
I have the same issue as well.

**Comment by jpsim** [comment]
Please file an issue with steps to reproduce if you need help.

**Comment by farzadshbfn** [comment]
I have the same prob too. It was working fine in `0.18.1`. I updated to `0.22`, and this feature is broken
it pops up warnings in my Pods, and I additionally added a `swiftlint autocorrect` in my scripts too, so i guess it changed some of the Pods projects codes as well

**Comment by ankitjaiswal1994** [comment]
Hello Everyone,

Make sure .swiftlint.yml file must be on the root path of your project.

**Comment by AdityaJha94** [comment]
How to ignore SwiftPackageManager from SwiftLint?
